src/prof/src/jemalloc.rs

```diff
@@ -18,174 +18,24 @@
 //! (1) Turn jemalloc profiling on and off, and dump heap profiles (`PROF_CTL`)
 //! (2) Parse jemalloc heap files and make them into a hierarchical format (`parse_jeheap`)
 
```

Comment:
The comment here is now outdated, as this module exports neither `PROF_CTL` nor `parse_jeheap`.